### PR TITLE
added the popup

### PR DESCRIPTION
**Pull request by Mov305**
New features
- Add the popup
- Implemented the functionality of the popup;
- Tested the linters
- Tested the popup

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -31,3 +31,82 @@ document.addEventListener('scroll', () => {
...
+    P2List.appendChild(listItem);
+  });
+  PopUp.classList.toggle('hidden');
+};
+
+cards.forEach((ele, i) => {
+  ele.addEventListener('click', () => {
+    makePopup(projectArray[i]);
+  });
+});
```
Comment (Ugboaja-Uchechi) [inline script.js:112]: - [ ] Great job 👍 . Immediately your website loads the pop-up displays by default. It is meant to display only when the user clicks on the button please fix this bug 🙏

File: style2.css [hunk 2]
```diff
@@ -15,6 +15,17 @@
   margin: 50px 28px;
 }
 
+#popup-demo > a {
+  flex-grow: 1;
+  padding: 10px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  display: flex;
```
Comment (Ugboaja-Uchechi) [inline style2.css:24]: - [ ] Kindly make sure your details pop-up design is as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Mov305, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- Repository looks professional 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — APPROVED** [review]
Hi @Mov305,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion
 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**